src/main/java/storagesystem/viewcontroller/settings/teamPage.fxml

```diff
@@ -0,0 +1,65 @@
+                  <Insets />
+               </opaqueInsets>
+            </TextField>
+            <Button layoutX="859.0" layoutY="261.0" mnemonicParsing="false" onAction="#removeMemberButtonPressed" prefHeight="32.0" prefWidth="115.0" styleClass="inAppButton" text="Remove user" textAlignment="CENTER" />
+            <Button fx:id="saveTeamButton" contentDisplay="BOTTOM" layoutX="712.0" layoutY="580.0" mnemonicParsing="false" onAction="#saveTeam" prefHeight="70.0" prefWidth="126.0" styleClass="inAppButton" text="Save" textAlignment="CENTER" AnchorPane.bottomAnchor="50.0" AnchorPane.rightAnchor="50.0" />
+            <Label layoutX="39.0" layoutY="23.0" text="Edit Team" textFill="WHITE" AnchorPane.leftAnchor="14.0" AnchorPane.topAnchor="0.0">
+               <font>
+                  <Font size="61.0" />
+               </font>
+            </Label>
+            <Text fill="WHITE" layoutX="689.0" layoutY="136.0" strokeType="OUTSIDE" strokeWidth="0.0" styleClass="bigLabel" text="Manage Members:" underline="true" />
+            <TextArea fx:id="settingsTeamContractInput" layoutX="214.0" layoutY="277.0" prefHeight="302.0" prefWidth="337.0" promptText="Write your team's standard terms and conditions here..." wrapText="true" />
+            <Text fill="WHITE" layoutX="63.0" layoutY="205.0" strokeType="OUTSIDE" strokeWidth="0.0" styleClass="bigLabel" text="Team Name:" underline="true" />
+            <Text fill="WHITE" layoutX="76.0" layoutY="303.0" strokeType="OUTSIDE" strokeWidth="0.0" styleClass="bigLabel" text="Terms and conditions:" underline="true" wrappingWidth="125.99998712539673" />
+            <Button fx:id="cancelTeamButton" layoutX="44.0" layoutY="615.0" mnemonicParsing="false" onAction="#cancelTeamButtonPressed" prefHeight="70.0" prefWidth="126.0" styleClass="inAppButton" text="Cancel" AnchorPane.bottomAnchor="50.0" AnchorPane.leftAnchor="50.0" />
+         </children>
+      </AnchorPane>
+      <AnchorPane fx:id="teamAnchorPane" prefHeight="700.0" prefWidth="1000.0" styleClass="background">
+         <children>
+            <Label text="Team:" textFill="WHITE" AnchorPane.leftAnchor="14.0" AnchorPane.topAnchor="0.0">
+               <font>
+                  <Font size="61.0" />
+               </font>
+            </Label>
+            <ChoiceBox fx:id="settingsChooseTeamInput" layoutX="764.0" layoutY="291.0" prefWidth="150.0" />
+            <Text fill="WHITE" layoutX="757.0" layoutY="265.0" strokeType="OUTSIDE" strokeWidth="0.0" styleClass="bigLabel" text="Choose a team:" underline="true" />
+            <Button fx:id="editTeamButton" layoutX="866.0" layoutY="574.0" mnemonicParsing="false" onAction="#editTeamButtonPressed" prefHeight="70.0" prefWidth="126.0" styleClass="inAppButton" text="Edit team" AnchorPane.bottomAnchor="50.0" AnchorPane.rightAnchor="50.0" />
+            <TextArea fx:id="teamToCTextArea" editable="false" layoutX="332.0" layoutY="240.0" prefHeight="325.0" prefWidth="337.0" wrapText="true" />
```

Comment:
Feels like this should look better. Make the background same as the anchorpane and change text color to white